operations/spire/Restore_Spire_Postgres_without_a_Backup.md

```diff
@@ -93,7 +88,7 @@ The CSM release tarball is required as it contains the Spire helm chart.
    ```
 
 7. Validate that the manifest.yaml file only contains chart information for Spire,
-and that the sources charts location points to the directory the helm chart was extracted from to prepend to /helm.
+   and that the sources charts location points to the directory the helm chart was extracted from to prepend to /helm.
```

Comment:
I don't know if this is what the sentence meant, but as it was written, it did not make much sense. 
```suggestion
   and that the source charts location points to the directory that the helm chart was extracted from.
```